alphadia/outputtransform/quantification/quant_builder.py

```diff
@@ -147,68 +182,91 @@ def filter_frag_df(
 
     def lfq(
```

Comment:
What do you think about having separate `quantselect_lfq` and `direct_lfq` functions here?
Having one general function which requires lot's of branching feels like more coupling, not less.